x-pack/legacy/plugins/ml/public/data_frame/pages/data_frame_new_pivot/route.ts

```diff
@@ -40,7 +38,7 @@ uiRoutes.when('/data_frames/new_transform', {
 });
 
 uiRoutes.when('/data_frames/new_transform/step/index_or_search', {
-  template: indexOrSearchTemplate,
+  template: '<ml-index-or-search></ml-index-or-search>',
```

Comment:
absolute nit pick, this could be `'<ml-index-or-search />'`
the same with the other templates in this PR